Tests/Integration/MSFT_xIISServerLevelIntegration.Tests.ps1

```diff
@@ -1,53 +1,90 @@
 ﻿######################################################################################
 # Integration Tests for DSC Resource for IIS Server level defaults
-# 
+#
 # These tests change the IIS server level configuration but roll back the changes at the end
 # so they should be save to run.
-# Run as an elevated administrator 
+# Run as an elevated administrator
 # At this time, we don't have tests for all changable properties, but it should be easy to add more tests.
 ######################################################################################
 
-$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+# should check for the server OS
+if($env:APPVEYOR_BUILD_VERSION)
```

Comment:
Could we change this code to something like this?:
```powershell
If (@(Get-WindowsOptionalFeature -Online -FeatureName 'IIS-WebServer' `
	| Where-Object -Property State -eq 'Disabled').Count -gt 0)
{
	If ((Get-CimInstance Win32_OperatingSystem).ProductType -eq 1)
	{
		# Desktop OS
		Enable-WindowsOptionalFeature -Online -FeatureName 'IIS-WebServer'
	} Else {
		# Server OS
                Install-WindowsFeature -IncludeAllSubFeature -IncludeManagementTools -Name 'Web-Server'
	}
}
```

It should then work on Server and Desktop OS's (in AppVeyor or not) - regardless if IIS is installed or not. Previous version would fail if run on a non-AppVeyor machine that does not have IIS installed.